Raycasting/src/LoopingSound.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "yse.hpp"
+#include <string>
+#include <array>
+
+class LoopingSound {
+public:
+	LoopingSound(const std::string filepath);
+
+	void setVolume(float volume);
+	void setPlaybackSpeed(float speed);
+
+	void play2D();
+	void play3D();
+	void setPosition(float x, float y, float z);
+	void setPositionRelitive(std::array<float, 2> observerPos, float observerFacingDirection, std::array<float, 3> soundPos);
```

Comment:
maybe change this to setPosition, just overload the inputs, add appt documentation to .cpp file

Reply:
Can documentation be added to implementation files? I can't remember... might have to use:
/** doc */

Reply:
no need, each overloaded function got its own documentation 😉